### DISCUSSION TITLE
FYI - Best Practices paper on ML training data

### DISCUSSION THREAD

**Discussion by thareUSGS** [pds-data-dictionaries/ldd-ml#12] [Ideas]
Nothing to do here but I ran across a best practices paper for training data sets including some recommended metadata attributes and format recommendations which we may find useful here. 
https://www.aireo.net/aireo-training-dataset-specification/

For testing ML methods, the community will need access to many different types of training data from craters, boulders, polygons, linea, etc.

**Comment by myche** [comment]
This is really interesting ... and a great find, @thareUSGS .

**Comment by stevenlujpl** [comment]
@thareUSGS Thanks for sharing the link. We (@sarabond ) will take a look and see how we can incorporate it into our ML LDD.

**Comment by sarabond** [comment]
Thanks so much for finding this and sharing, @thareUSGS ! I'm in the process of adding new keywords to the ML LDD with guidance from Steven, so this is great timing.
